frontend/assets/js/calculator.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', () => {
+    init();
+});
+
+function init() {
+    document.getElementById('split').addEventListener('click', () => {
+        split();
+    });
+
+    document.getElementById('clear').addEventListener('click', () => {
+        clear();
+    });
+}
+
+function split() {
+    let amount = document.getElementById('amount');
+    let people = document.getElementById('people');
+
+    let message = document.getElementById('message');
+    let result = 'Each person will pay you = ';
+
+    if(people.value === '' && amount.value === '') {
+        message.textContent = 'Please Enter valid value';
+        return;
+    }
+
+    if(people.value != 0) {
+        result += amount.value / people.value;
+    }
+    else {
+        result += amount.value;
+    }
+
+    message.textContent = result;
+}
+
+function clear() {
+    let amount = document.getElementById('amount');
+    let people = document.getElementById('people');
+    let message = document.getElementById('message');
```

Comment:
The declaration has been done in split function as well. You might be able to define it above those functions, therefor you will not need to call them each time.